install.sh

```diff
@@ -288,6 +288,21 @@ meshcfg="$(cat << EOF
     "MaxInvalidLogin": { "time": 5, "count": 5, "coolofftime": 30 }
   },
   "domains": {
+    "domaindefaults":{
+      "passwordRequirements": {
+        "min": 8,
+        "max": 128,
+        "upper": 1,
+        "lower": 1,
+        "numeric": 1,
+        "nonalpha": 1,
+        "reset": 90,
+        "force2factor": false,
+        "oldPasswordBan": 5,
+        "banCommonPasswords": false,
```

Comment:
Is there a reason not to make this true?

Reply:
No, I will fix it, give me a second